packages/ListBox/stories/examples/Showcase.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { action } from "@storybook/addon-actions";
+import { boolean, text } from "@storybook/addon-knobs";
+import { Rule } from "storybook/assets/styles/common.styles";
+import * as characters from "../fixtures/characters";
+import ListBox from "../../src";
+
+const getKnobs = () => ({
+  hasImplicitAll: boolean("hasImplicitAll", false),
+  isDisabled: boolean("isDisabled", false),
+  placeholder: text("placeholder", "Select..."),
+  showFilter: boolean("Show Filter", false),
+  showFooter: boolean("Show Footer", false),
+});
+
+export default function Showcase() {
+  const { showFilter, showFooter, ...listBoxProps } = getKnobs();
+
+  const ListBoxChildren = (
+    <>
+      {showFilter && <ListBox.Filter />}
+      {characters.villians}
+    </>
+  );
+
+  return (
+    <>
+      <h3>Single</h3>
```

Comment:
`<StoryHeader>` please!
Also, maybe we can be a bit more descriptive? "Single-select – default" / "Multi-select – `isMulti`" or something like that? 

I think it would be good to indicate that the `isMulti` prop is the difference here.  Maybe it even works if `isMulti` is a knob...?

Reply:
isMulti won't work as a knob unfortunately as the component doesn't re-render